docs/index.html

```diff
@@ -89,32 +89,32 @@ <h1 id="commands">Commands:</h1>
 betterTeams.standard
 Commands all users have access to:</p>
 <ul>
-  <li>/team join <team> - Joins the specified team</team></li>
-  <li>/team create <team> - Creates a new team with that name</team></li>
+  <li>/team join [team] - Joins the specified team</li>
+  <li>/team create [team] - Creates a new team with that name</li>
   <li>/team leave - Leaves the current team</li>
   <li>/team home - Teleports to the teams home location</li>
   <li>/team chat [message] - If a message is included, it will send a single message to the team chat, if no message is included the player will be moved to the team chat (all further messages will go there)</li>
   <li>/team info [team/player] - Provides information on the team or the team that the specified player is in.
 */team help - Provides a personalised help page, which will only show the commands that the user has access to
 Commands that admins and owners of teams have access to:</li>
-  <li>/team invite <player> - Invites the specified player to the team</player></li>
-  <li>/team kick <player> - Kicks the specified player from the team (players can only kick members a lower rank than their own)</player></li>
-  <li>/team ban <player> - Bans a player from the team (players can only ban users a lower rank than their own)</player></li>
-  <li>/team unban <player> - Allows the specified player to join back into the team.</player></li>
+  <li>/team invite [player] - Invites the specified player to the team</li>
```

Comment:
These should not be changed, the MC standard is that <> denotes required and [] denotes optional

Reply:
probably should switch to &gt etc. or whatever it is (I've already forgotten)

Reply:
yeah indeed, then it should be &gt; -- &lt;

Reply:
Fixed this. Can you check as it's getting late and I'm not my best anymore.